airflow/executors/kubernetes_executor.py

```diff
@@ -267,6 +267,15 @@ def __init__(self):
             self.kube_client_request_args = {}
         self._validate()
 
+    # pod security context items should return integers
+    # and only return a blank string if contexts are not set.
+    def _get_security_context_val(self, scontext):
+        val = configuration.get(self.kubernetes_section, scontext)
+        if len(val) == 0:
```

Comment:
Is there a scenario where val can be None?
If so it will raise exception `TypeError: object of type 'NoneType' has no len()`

Reply:
No, not given that the default config has `run_as_user =` in it.